conf/airframes/ENAC/quadrotor/anton_indi_aruco.xml

```diff
@@ -267,9 +245,9 @@
   </section>
 
   <section name="SIMULATOR" prefix="NPS_">
-    <define name="ACTUATOR_NAMES" value="nw_motor, ne_motor, se_motor, sw_motor" type="string[]"/>
-    <define name="JSBSIM_MODEL" value="simple_x_quad_ccw" type="string"/>
-    <define name="SENSORS_PARAMS" value="nps_sensors_params_default.h" type="string"/>
+    <define name="ACTUATOR_NAMES" value="ne_motor, se_motor, sw_motor, nw_motor" type="string[]"/>
```

Comment:
why is the order different than most other airframes?

Reply:
Because the motors are not declared in the same order in most of our airframes compared to quadrotors in examples. I don't know why it is like this, but I preferred to change the order of actuators in sim (that was broken) instead of the order in `servos` section (that was working on real platforms).